src/app.tsx

```diff
@@ -60,6 +68,35 @@ function AppRoutes() {
     { path: "/settings/projects/:projectId/plugins", element: <PluginSettings /> },
     { path: "/plugin-editor", element: <PluginEditor /> },
     { path: "/graphql", element: process.env.NODE_ENV !== "production" && <GraphQLPlayground /> },
+    // redirections for breaking changes in urls
+    {
+      path: "/settings/workspace/:teamId",
+      element: <Redirect to="/settings/workspaces/:teamId" />,
```

Comment:
Are these really necessary? I feel like we are making things more complex here to avoid having to update URLs in probably just the documentation website.
I think remember seeing @rot1024 asked for these, so maybe he can shed some light here?

Reply:
As we talked i want to provide the redirections at least for the next release so that users can be noticed about the breaking changes from release note -> update outdated urls in bookmark or docs (if any) and won't facing a 404 directly. The codes from @rot1024 also shows he want to keep it maybe. So i just mark the coversation as resolved now and let me know if you want to discuss more :)